Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/DefaultObjectSrg.azsli

```diff
@@ -40,16 +40,43 @@ ShaderResourceGroup ObjectSrg : SRG_PerObject
     //! Reflection Probe (smallest probe volume that overlaps the object position)
     struct ReflectionProbeData
     {
-        float3 m_aabbPos;
-        float3 m_outerAabbMin;
-        float3 m_outerAabbMax;
-        float3 m_innerAabbMin;
-        float3 m_innerAabbMax;
+        row_major float3x4 m_modelToWorld;
+        row_major float3x4 m_modelToWorldInverse; // does not include extents
```

Comment:
It seems this matrix is similar with SceneSrg::m_objectToWorldInverseTransposeBuffer

Can you use SceneSrg::m_objectToWorldInverseTransposeBuffer
 to construct the matrix for GetReflectionProbeWorldMatrixInverse() function instead of add extra matrix here?

Reply:
That buffer is for the draw call, I need separate transforms for the reflection probe.